care/utils/queryset/filters.py

```
@@ -0,0 +1,42 @@
+
+    Examples:
+        # Sort patients by priority status: ['critical', 'urgent', 'stable']
+        patients = Patient.objects.annotate(
+            priority_order=sort_index('status', ['critical', 'urgent', 'stable'])
+        ).order_by('priority_order')
+    """
+    if not order:
+        return models.Case()
+
+    if not isinstance(order, list):
+        raise ValueError("Order must be a list")
```

Ruff (0.12.2)

Prefer `TypeError` exception for invalid type

(TRY004)